store/tikv/2pc.go

```diff
@@ -564,7 +564,9 @@ func (c *twoPhaseCommitter) prewriteSingleBatch(bo *Backoffer, batch batchKeys)
 			locks = append(locks, lock)
 		}
 		start := time.Now()
-		msBeforeExpired, err := c.store.lockResolver.ResolveLocks(bo, locks)
+
+		// Prewrite pessimistic transaction should not meet lock?
+		msBeforeExpired, _, err := c.store.lockResolver.ResolveLocks(bo, c.startTS, locks)
```

Comment:
It will meet non-pessimistic locks. https://github.com/tikv/tikv/blob/master/src/storage/mvcc/txn.rs#L200